tests/test_filters.py

```diff
@@ -118,3 +121,18 @@ def test_capitalize_first_for_non_strings():
     assert capitalize_first([{'list': 'of'}, 'things']) == [{'list': 'of'}, 'Things']
     assert capitalize_first({'this': 'thing'}) == {'this': 'thing'}
     assert capitalize_first('https://www.example.com') == 'https://www.example.com'
+
+
+@pytest.mark.parametrize("bool", (False, True,))  # Shouldn't matter
+def test_preserve_line_breaks(bool):
```

Comment:
`bool` is a builtin, which makes the calls a bit confusing without seeing the test declaration. Could we call it something like `autoescape_flag` to disambiguate?

Also, why are we testing this for both cases if we're not checking the Markup return type?